### ISSUE TITLE
Extension breaks on double quoted strings surrounded by apostrophes

### ISSUE THREAD

**Issue by ImRodry** [meganrogge/template-string-converter#67]
I just had the following edge case happen to me:
Original string
`"Couldn't find user ${user}. Are you sure they're on the server?"`
Upon typing ${user}, the extension converted the two apostrophes (`'`) in "Couldn't" and `they're` to template strings, instead of converting the double quotes which were the ones declaring the current string, leaving me with this:
`"Couldn`t find user ${user}. Are you sure they`re on the server?"`
I understand this is an edge case and may be hard to fix, I'm just reporting it here so you're aware of it.

**Comment by meganrogge** [comment]
Yeah it's pretty hard to account for cases like this. The setting `template-string-converter.convertOutermostQuotes` would help you though

**Comment by Sheraff** [comment]
If this is not the default option, what is the downside of activating that? Is it a few edge-cases? Some specific languages with intricate quote syntax? Just a matter of preference?

**Comment by meganrogge** [comment]
There have been requests to convert the innermost quotes. It's a matter of preference, thus the setting

**Comment by talks2much** [comment]
> Is it a few edgecases?

I haven't spoted any cases. it seem to work ideally?

I also completely don't understand why this setting is disabled by default? When its 'false' it doesn't convert the string itself to
template string in provided cases because of syntax of JS, thats why I think It shouldnt be out-bf-box behavior. This
can be surprising for new users.

@rneganrogge or at least I think it'd be better to put this setting to top in readme and highlight default value (or even better to change default if possble)
